client/src/containers/delete-user/component.tsx

```diff
@@ -0,0 +1,66 @@
+import { useState } from 'react';
+import { ExclamationIcon } from '@heroicons/react/outline';
+import toast from 'react-hot-toast';
+
+import { Button } from 'components/button';
+import Modal from 'components/modal/component';
+import { useDeleteUser } from 'hooks/users';
+
+type DeleteUserProps = {
+  userName: string;
+  id: string;
+};
+
+const DeleteUser = ({ id, userName }: DeleteUserProps) => {
```

Comment:
```suggestion
const DeleteUserModal = ({ id, userName }: DeleteUserProps) => {
```
I'd prefer to add more info in the component naming.